src/highlight.cpp

```diff
@@ -468,7 +468,7 @@ bool autosuggest_validate_from_history(const history_item_t &item,
     }
 
```

Comment:
> Fix: #7582

The commit message should rather say `Fixes #7582` or `Closes #7582`, otherwise the issue won't be closed automatically